.github/workflows/publish.yml

```diff
@@ -6,12 +6,15 @@ permissions:
 on:
   release:
     types: [published]
+  # To manually trigger a rebuild of the API docs if needed
+  workflow_dispatch:
 
 jobs:
   publish:
     name: Langium Publish
     runs-on: ubuntu-latest
     timeout-minutes: 20
+    if: ${{ github.event_name == 'release' }}
```

Comment:
I don't know if we want to constrain the publish job like this. We didn't need it before, and I think we may want to run this under different circumstances than this constraint would allow.

Reply:
Because we didn't have any other trigger before, but now I added the `workflow_dispatch` trigger to allow updating the docs manually even before a release. Now that I think about it, I'm not sure if this would actually ever be used

Reply:
Hmm, yeah I'm still not sure about this one. @msujew thoughts here?

Reply:
I think it makes more sense to have a separate `workflow_dispatch` job for the API docs publishing - right now, the release publishing process is broken due to expired credentials.

Reply:
By "separate job", do you mean an entirely separate .yml file or just the additional trigger here?

Reply:
A separate yml file :)

Reply:
done